python/paddle/fluid/layers/nn.py

```diff
@@ -10652,3 +10652,52 @@ def tree_conv(nodes_vector,
     else:
         pre_activation = out
     return helper.append_activation(pre_activation)
+
+
+def npair_loss(anchor, positive, labels, l2_reg=0.002):
+    '''
+  **Npair Loss Layer**
+  
+  Read `Improved Deep Metric Learning with Multi class N pair Loss Objective <http://www.nec-labs.com/uploads/images/Department-Images/MediaAnalytics/papers/nips16_npairmetriclearning.pdf>`_ .
+  
+  Npair loss requires paired data. Npair loss has two parts: the first part is L2 
+  regularizer on the embedding vector; the second part is cross entropy loss which
+  takes the similarity matrix of anchor and positive as logits.
+
+  Args:
+    anchor(Variable): embedding vector for the anchor image. shape=[batch_size, embedding_dims]
+    positive(Variable): embedding vector for the positive image. shape=[batch_size, embedding_dims]
+    labels(Variable): 1-D tensor. shape=[batch_size]
+    l2_reg(float32): L2 regularization term on embedding vector, default: 0.002
+
+  Returns:
+    npair loss(Variable): return npair loss, shape=[1]
+
+  Examples:
+    .. code-block:: python
+
+       npair_loss = fluid.layers.npair_loss(anchor, positive, labels, l2_reg)
```

Comment:
```suggestion
       npair_loss = fluid.layers.npair_loss(anchor, positive, labels, l2_reg)
```
可否在示例里面添加关于定义anchor, positive, labels, l2_reg的语句并加以简单注释，这样示例
才能真正起到效果。

Reply:
好的